api/envoy/extensions/filters/http/jwt_authn/v3/README.md

```diff
@@ -30,6 +30,18 @@ If a custom location is desired, `from_headers` or `from_params` can be used to
 
 If a JWT is valid, its payload will be passed to the backend in a new HTTP header specified in `forward_payload_header` field. Its value is base64url-encoded JWT payload in JSON.
 
+## Copy Validated JWT Claims to HTTP Request Headers
```

Comment:
Please move this new section to `docs/root/configuration/http/http_filters/jwt_authn_filter.rst`.  This doc will be removed.

Reply:
should I delete the content from `README.md` as well ?

Reply:
Yes, I am going to remove this doc in https://github.com/envoyproxy/envoy/pull/23923